Gauntlet.toml

```diff
@@ -1406,31 +1406,16 @@ document = "broadinstitute/palantir-workflows:/BenchmarkSVs/BenchmarkSVs.wdl"
 message = "BenchmarkSVs.wdl:457:40: error: a placeholder cannot have more than one option"
 permalink = "https://github.com/broadinstitute/palantir-workflows/blob/f8833a0000cd441a3cc4be2718876a1433cb21ef/BenchmarkSVs/BenchmarkSVs.wdl/#L457"
 
-[[diagnostics]]
-document = "broadinstitute/palantir-workflows:/BenchmarkSVs/BenchmarkSVs.wdl"
-message = "BenchmarkSVs.wdl:457:49: error: cannot coerce type `Array[Int]` to `String`"
```

Comment:
Adding a comment here for my own purposes as the removal of these diagnostics looks incorrect to me.

I'll revisit once I've made it further into the changes.

Reply:
Okay, I understand what's going on here and why placeholders was changed to be an iterator in the AST.

We're already emitting a diagnostic here that there can't be more than one placeholder; we're not allowing the coercion from the array because the first option is `default` and so we don't consider the `sep` option.

I'm not convinced it is worth fixing, to be honest.